jupyterhub/tests/test_proxy.py

```diff
@@ -184,6 +184,18 @@ def test_check_routes(app,  username, disable_check_routes):
     # check that before and after state are the same
     assert before == after
 
+def test_restore_routes(app, username):
```

Comment:
This needs `@pytest.mark.gen_test` to make sure it's run as an async test

Reply:
Additionally, `username` isn't a fixture, so move that down inside the test, e.g.


```python
username = 'kshitija'
```